gateway/1.4/modules/ROOT/pages/_partials/task-config-liveness-probe.adoc

```diff
@@ -0,0 +1,69 @@
+//tag::intro1[]
+
+Flex Gateway includes the following CLI command to test gateway health. You can run this command manually on any Flex Gateway:
+
+[source,ssh]
+----
+flexctl probe --check=liveness
+----
+
+This command returns either an exit code of `0` for a healthy Flex Gateway or `1` for an unhealthy Flex Gateway.
```

Comment:
```suggestion
If the Flex Gateway instance is healthy, the command returns an exit code of `0`. Otherwise it returns a `1`.
```
Shortening sentences.